### PR TITLE
bracket validated

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/src/main/java/Java/multibracketvalidation/multiBracketValidation.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package Java.multibracketvalidation;
+
+import Java.stacksandqueues.Stacks;
+
+import java.util.Stack;
+
+public class multiBracketValidation {
+
+    static boolean pairsAreMatching(char uno, char dos) {
```
Comment (evanslaton) [inline challenges/src/main/java/Java/multibracketvalidation/multiBracketValidation.java:9]: This is great!
